src/wasm-binary.h

```diff
@@ -37,7 +37,8 @@ namespace wasm {
 
 enum {
   // the maximum amount of bytes we emit per LEB
-  MaxLEB32Bytes = 5
+  MaxLEB32Bytes = 5,
+  MaxDataSegments = 100 * 1000 // a hard limit wasm VMs impose
```

Comment:
Is the limit in VMs `100*1000`, or `100*1024`?
Is there a reference we can link to for this, because all I see on https://webassembly.github.io/spec/core/appendix/implementation.html?highlight=implementation%20limit is "implementations may impose some limit"

Reply:
I also can't find a proper reference for it. I figured I'd set the limit to the lower of the two interpretations ;)

Reply:
https://github.com/WebKit/webkit/blob/master/Source/JavaScriptCore/wasm/WasmLimits.h

Reply:
Is there an official link for the Web as a whole?

Reply:
The official discussion is here: https://github.com/WebAssembly/spec/issues/607